app/frontend/src/components/views/project-view/components/TaskInfo.tsx

```diff
@@ -19,6 +31,62 @@ interface Props {
  * @param Props.task - current task
  */
 const TaskInfo: React.FC<Props> = ({ projectTitle, task }) => {
+  const [statusesOptions, setStatusesOptions] = useState<DropdownItem[]>([]);
+  const [statuses, setStatuses] = useState<StatusType[]>([]);
+
+  useEffect(() => {
+    (async function () {
```

Comment:
please explain this code section. Why to you request statuses everytime the popup is open?